src/neuroconv/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -163,7 +163,8 @@ def run_conversion(
             If the processing module specified by module_name does not exist, it will be created with this description.
             The default description is the same as used by the conversion_tools.get_module function.
         compression: str, optional
-            Compression strategy to use for HFDataIO. For full list of currently supported filters, see
+            Compression strategy to use for :py:class:`hdmf.backends.hdf5.h5_utils.H5DataIO`. For full list of currently
+            supported filters, see
```

Comment:
So, if I understand correclty HFDataIO should be displayed as a link to the relevant class in hdmf library, correct?

It does not seem to be working

![image](https://user-images.githubusercontent.com/6429509/182303506-d1fe7bab-dcb1-4d91-b5e9-3405dd97d90b.png)

Reply:
It is for me:

<img width="690" alt="image" src="https://user-images.githubusercontent.com/844306/182369495-54d551b4-b7a4-46eb-b12d-ebca3a0de15a.png">

Reply:
Yes, I don't know what happened. I double checked and it seems to be working now from the link (!). So, it is working for me as well.